samples/nrf9160/at_cmd_parser/src/main.c

```diff
@@ -0,0 +1,344 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+
+#include <at_cmd_parser/at_cmd_parser.h>
+#include <at_cmd_parser/at_params.h>
+
+const char *test1 = "+CEREG: 2,\"76C1\",\"0102DA04\", 7";
```

Comment:
So I get an impression it's more of a set of unit tests than an actual sample. Could we move it into `fw-nrfconnect-nrf/tests` directory then, and use `ztest` framework to execute them? See https://github.com/NordicPlayground/fw-nrfconnect-nrf/tree/master/tests/aws_fota_json for example how this could be done.

Reply:
Look at my other comment. I don't have any strong options, but I think it may be useful for users to see how the library treat different use-cases.